src/libcalls.jl

```diff
@@ -0,0 +1,48 @@
+function libsvm_get_max_threads()
+    return ccall((:svm_get_max_threads, libsvm), Cint, ())
+end
```

Comment:
Actually this function is simple enough.
```diff
diff --git a/src/libcalls.jl b/src/libcalls.jl
index bae5662..eb0b1e5 100644
--- a/src/libcalls.jl
+++ b/src/libcalls.jl
@@ -1,6 +1,4 @@
-function libsvm_get_max_threads()
-    return ccall((:svm_get_max_threads, libsvm), Cint, ())
-end
+libsvm_get_max_threads() = ccall((:svm_get_max_threads, libsvm), Cint, ())
 
 function libsvm_set_num_threads(n::Integer)
     ccall((:svm_set_num_threads, libsvm), Cvoid, (Cint,), n)
```

Reply:
You are right. However, would you be willing to leave it as it is now? Because `libsvm_get_max_threads() = ccall((:svm_get_max_threads, libsvm), Cint, ())` is too long to be on a single line, and it is not very aesthetically pleasing when split into multiple lines :smile:

Reply:
:ok_hand: